docs/architecture/security/principles/01-servers-are-not-trusted.mdx

```diff
@@ -0,0 +1,37 @@
+# P01 - Servers are not trusted
+
+Clients do not need to trust Bitwarden server or web infrastructure to be good actors in supplying
+data to keep an account's encrypted content secure. In other words, the special status of providing
```

Comment:
suggestion: do you think "in supplying data" might be redundant? Might also be unnecessarily restrictive
```suggestion
Clients do not need to trust Bitwarden server or web infrastructure to be good actors to keep an account's encrypted content secure. In other words, the special status of providing
```

Reply:
Would this include serving a malicious web vault then? Is that a guarntee we want to / can give?

Reply:
Ideally, it definitely would be able to guarantee web vault code. I don't know if we'd ever achieve that goal, but not knowing how to do it right now doesn't mean it's not inline with a principle we have.

@coroiu I agree that's better both because it removes weird caveats and is clearer in intent :shipit: